### DISCUSSION TITLE
SWF Support?

### DISCUSSION THREAD

**Discussion by rdrehmer** [ccseer/Seer#380] [General]
Hi!
Any chance of getting preview for SWF files?
Flash might be dead for web, but not for animation yet :)

Thanks!

**Comment by ccseer** [comment]
I just checked it.

It can be supported by plugin in html view. 
But Seer can't support it now, I disabled the time consuming operation.

**Comment by rdrehmer** [comment]
Oh, I see. That's unfortunate.
I usually open SWF files with Media Player Classic (or VLC), so they play like a normal video. Of course it ignores all interactivity, as well as doesn't recognize stage color. But it doesn't matter in my particular case, since I just work with animation, not web.

I don't really know about this, but it seems that there is a way to read it just like a video, instead of a full feature html plugin. Does it make sense or not at all?

**Comment by ccseer** [comment]
Hi

The OIT plugin seems suuport Flash
https://docs.oracle.com/outsidein/853/oit/supportdocs/ds_oitfiles_8_5_3.pdf

Maybe you can add the extension manually to see if it works.
